File: Server/Web.Domain/Entities/ContractEntity.cs
namespace Api.Domain.Entities
{
    public class ContractEntity
    {
        public ulong Id { get; set; }
        public string Author { get; set; } = string.Empty;
        public string Name { get; set; } = string.Empty;
        public string Description { get; set; } = string.Empty;
        public DateTime CreatedDate { get; set; }
        public DateTime? UpdatedDate { get; set; } = null;
    }
}

File: Web/src/entities/ContractEntity.ts
export class ContractEntity {
      id: number = 0;
      author: string = '';
      name: string = '';
      description: string = '';
      createdDate: string = Date.toString();
      updatedDate: string = Date.toString();
      get descriptionShort(): string { 
        return (this.description.length > 20) ? this.description.substring(0,17)+'...' : this.description; 
      }
    }